modules/tensor_mechanics/src/materials/ADPowerLawCreepStressUpdate.C

```diff
@@ -79,6 +79,17 @@ ADPowerLawCreepStressUpdate::computeDerivative(const ADReal & effective_trial_st
   return creep_rate_derivative * _dt - 1.0;
 }
 
+ChainedADReal
+ADPowerLawCreepStressUpdate::computeResidualAndDerivative(
+    const ChainedADReal & effective_trial_stress, const ChainedADReal & scalar)
+{
+  const ChainedADReal stress_delta =
+      effective_trial_stress - ChainedADReal(_three_shear_modulus, 0) * scalar;
+  const ChainedADReal creep_rate = ChainedADReal(_coefficient * _exponential * _exp_time, 0) *
+                                   std::pow(stress_delta, _n_exponent);
+  return creep_rate * _dt - scalar;
+}
+
```

Comment:
This is the only stress update class I converted. All the other ones should still reserve their original behavior.

It looks ugly -- I had to explicit construct `ChainedADReal`, as otherwise operators like `operator*` are not defined over `ChainedADReal` and `ADReal`. I don't have the time to overload all those operators, so this is the show-stopper for now.